src/scss/components/_verification-badge.scss

```diff
@@ -2,7 +2,7 @@
 
   background-color: map-get($site-palette, 'site-white');
   border: units(1px) solid map-get($site-palette, 'site-green');
-  border-radius: $theme-border-radius-xxl;
```

Comment:
"XXL" is not a size for border radius in USWDS. This usage must have been intended to consider it as a custom addition. But it's not as simple as adding just the variable, since the size tokens are expected to be usable from within e.g. [utility mixins](https://designsystem.digital.gov/utilities/border/#utility-radius). Code like `@include u-radius('xxl');` doesn't work otherwise. I'm not sure if it's intended to be possible to override these from the [default radius settings](https://github.com/uswds/uswds/blob/0b469e41fa9f6a947ffcd546b79f3a8aec0002ce/src/stylesheets/core/_variables.scss#L598-L617). In any case, since it wasn't really working correctly aside from direct variable reference, and since we don't reference the variable in any of our other projects, I figured the easiest decision is to eliminate the variable and inline its value.

Reply:
+1 💊

Reply:
> In practice, this is likely a candidate for the ["pill" style radius](https://designsystem.digital.gov/utilities/border/#utility-radius) anyways 🤔 I may just update it to use that.

Updated in f8e1e63.